tutorials/understanding-group/index.md

```diff
@@ -205,7 +205,7 @@ As an exercise, please add Bob back in the group and go to the next section.
 ## Create a group policy
 
 Next you need to decide on a group policy. This defines how long a proposal can be voted on and how its outcome is calculated.
-Here you use the `ThresholdDecisionPolicy`. It defines the threshold that the tally of weighted _yes_ votes must reach in order for a proposal to pass. Each member's vote is weighted by its weight as defined in the group.
+Here you use the `ThresholdDecisionPolicy`. It defines the threshold that the tally of weighted *yes* votes must reach in order for a proposal to pass. Each member's vote is weighted by its weight as defined in the group.
```

Comment:
Is this Markdownlint too? Throughout the content we use `_` for italics.

Reply:
Okay, I can tweak the `markdownlint.json` to that then. Currently, it is set to `*`.